static/js/navigation/search.js

```diff
@@ -0,0 +1,41 @@
+import { navigation, secondaryNavigation } from "./elements";
+
+import closeAllNavigationItems from "./main";
+
+/**
+ * Handle the state of the search
+ */
+export function toggleSearch() {
+  const isOpen = navigation.classList.contains("has-search-open");
+
+  if (!isOpen) {
+    closeAllNavigationItems();
+    openSearch();
+  } else {
+    closeAllNavigationItems();
+  }
```

Comment:
The conditional expression can be refactored 
```suggestion
  closeAllNavigationItems();
  if (!isOpen) {
    openSearch();
  }
```

Reply:
nice :fire: